R-packages/evalcast/data-raw/county_label_lookup.R

```diff
@@ -0,0 +1,5 @@
+## code to prepare `county_label_lookup` dataset goes here
+county_label_lookup <- paste(
```

Comment:
Where is `county_label_lookup` used?

Reply:
I'm not sure where this came from. I must have screwed up the git. I think this script and the associated data file below should just be deleted.